bundles/UuidBundle/src/Resources/install/install.sql

```diff
@@ -1,7 +1,7 @@
 DROP TABLE IF EXISTS `uuids`;
 CREATE TABLE `uuids` (
     `uuid` CHAR(36) NOT NULL,
-    `itemId` int(11) unsigned NOT NULL,
+    `itemId` VARCHAR(50) NOT NULL,
```

Comment:
How did you determine how long the varchar can be?

Reply:
Based on the class definition ID column type..

Reply:
please see https://github.com/pimcore/pimcore/blob/f33ee7e25c596201668109648823c41a6d5dd127/bundles/InstallBundle/dump/install.sql#L77